samples/highcharts/demo/3d-column-stacking-grouping/demo.js

```diff
@@ -11,11 +12,11 @@ Highcharts.chart('container', {
     },
 
     title: {
-        text: 'Total fruit consumption, grouped by gender'
+        text: ' Electricity production in countries, grouped by continent'
     },
 
     xAxis: {
-        categories: ['Apples', 'Oranges', 'Pears', 'Grapes', 'Bananas'],
+        categories: ['2016', '2017', '2018', '2019', '2020'],
```

Comment:
Better use a linear axis here. Remove the categories and instead add something like this:

```js
plotOptions: {
    series: {
        pointStart: 2016
    }
}
```